app/controllers/comment_controller.php

```diff
@@ -0,0 +1,55 @@
+<?php
+class CommentController extends AppController
+{
+	CONST MAX_ITEM_PER_PAGE = 10;
+
+	public function view()
+	{
+		session_start();
```

Comment:
I have redirect_to_login() function po at validation_helpers.php which is called in every views po to check po if the user is currently logged in or not. But I'm planning to move it to my default layout po.